arch/arm/src/stm32l4/hardware/stm32l4_can.h

```diff
@@ -176,6 +179,60 @@
+#  define STM32L4_CAN2_ESR          (STM32L4_CAN2_BASE+STM32L4_CAN_ESR_OFFSET)
+#  define STM32L4_CAN2_BTR          (STM32L4_CAN2_BASE+STM32L4_CAN_BTR_OFFSET)
+
+#  define STM32L4_CAN2_TIR(m)       (STM32L4_CAN2_BASE+STM32L4_CAN_TIR_OFFSET(m))
+#  define STM32L4_CAN2_TI0R         (STM32L4_CAN2_BASE+STM32L4_CAN_TI0R_OFFSET)
+#  define STM32L4_CAN2_TI1R         (STM32L4_CAN2_BASE+STM32L4_CAN_TI1R_OFFSET)
+#  define STM32L4_CAN2_TI2R         (STM32L4_CAN2_BASE+STM32L4_CAN_TI2R_OFFSET)
+
+#  define STM32L4_CAN2_TDTR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_TDTR_OFFSET(m))
+#  define STM32L4_CAN2_TDT0R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDT0R_OFFSET)
+#  define STM32L4_CAN2_TDT1R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDT1R_OFFSET)
+#  define STM32L4_CAN2_TDT2R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDT2R_OFFSET)
+
+#  define STM32L4_CAN2_TDLR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_TDLR_OFFSET(m))
+#  define STM32L4_CAN2_TDL0R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDL0R_OFFSET)
+#  define STM32L4_CAN2_TDL1R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDL1R_OFFSET)
+#  define STM32L4_CAN2_TDL2R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDL2R_OFFSET)
+
+#  define STM32L4_CAN2_TDHR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_TDHR_OFFSET(m))
+#  define STM32L4_CAN2_TDH0R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDH0R_OFFSET)
+#  define STM32L4_CAN2_TDH1R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDH1R_OFFSET)
+#  define STM32L4_CAN2_TDH2R        (STM32L4_CAN2_BASE+STM32L4_CAN_TDH2R_OFFSET)
+
+#  define STM32L4_CAN2_RIR(m)       (STM32L4_CAN2_BASE+STM32L4_CAN_RIR_OFFSET(m))
+#  define STM32L4_CAN2_RI0R         (STM32L4_CAN2_BASE+STM32L4_CAN_RI0R_OFFSET)
+#  define STM32L4_CAN2_RI1R         (STM32L4_CAN2_BASE+STM32L4_CAN_RI1R_OFFSET)
+
+#  define STM32L4_CAN2_RDTR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_RDTR_OFFSET(m))
+#  define STM32L4_CAN2_RDT0R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDT0R_OFFSET)
+#  define STM32L4_CAN2_RDT1R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDT1R_OFFSET)
+
+#  define STM32L4_CAN2_RDLR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_RDLR_OFFSET(m))
+#  define STM32L4_CAN2_RDL0R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDL0R_OFFSET)
+#  define STM32L4_CAN2_RDL1R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDL1R_OFFSET)
+
+#  define STM32L4_CAN2_RDHR(m)      (STM32L4_CAN2_BASE+STM32L4_CAN_RDHR_OFFSET(m))
+#  define STM32L4_CAN2_RDH0R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDH0R_OFFSET)
+#  define STM32L4_CAN2_RDH1R        (STM32L4_CAN2_BASE+STM32L4_CAN_RDH1R_OFFSET)
+
+#  define STM32L4_CAN2_FMR          (STM32L4_CAN2_BASE+STM32L4_CAN_FMR_OFFSET)
+#  define STM32L4_CAN2_FM1R         (STM32L4_CAN2_BASE+STM32L4_CAN_FM1R_OFFSET)
+#  define STM32L4_CAN2_FS1R         (STM32L4_CAN2_BASE+STM32L4_CAN_FS1R_OFFSET)
+#  define STM32L4_CAN2_FFA1R        (STM32L4_CAN2_BASE+STM32L4_CAN_FFA1R_OFFSET)
+#  define STM32L4_CAN2_FA1R         (STM32L4_CAN2_BASE+STM32L4_CAN_FA1R_OFFSET)
+#  define STM32L4_CAN2_FIR(b,i)     (STM32L4_CAN2_BASE+STM32_CAN_FIR_OFFSET(b,i))
```

Comment:
```suggestion
#  define STM32L4_CAN2_FIR(b,i)     (STM32L4_CAN2_BASE+STM32L4_CAN_FIR_OFFSET(b,i))
```

Reply:
for matting is off too should be X + Y to X+Y

Reply:
@ASM3 please change STM32_CAN_FIR_OFFSET to STM32L4_CAN_FIR_OFFSET as Gustavo pointed and remove it from Draft